configs/next/deb/monolithic/buster_dh_change_dest.patch

```diff
@@ -1 +1 @@
-../../../15.0/deb/monolithic/buster_dh_change_dest.patch
\ No newline at end of file
+../../../16.0/deb/monolithic/buster_dh_change_dest.patch
```

Comment:
Are we still going to support Debian Buster for AT 16? If not, I suspect we don't need this file.

Reply:
I forgot this one too.